pkgs/top-level/all-packages.nix

```diff
@@ -16352,7 +16352,10 @@ with pkgs;
 
   ograc = callPackage ../development/tools/rust/ograc { };
 
+  opensmalltalk-vm = callPackage ../development/compilers/opensmalltalk-vm { };
```

Comment:
This set should also be flattened to expose all the VMs that are built as top-level packages.

Reply:
How do you recommend flattening the set? Should I prefix `opensmalltalk-vm-` to each name, or leave them as `squeak-cog-spur`, etc.? I prefer prefixing or not flattening, so that these packages are grouped in some way.